client/signup/steps/import-from/index.tsx

```diff
@@ -112,6 +113,22 @@ const ImportOnboardingFrom: React.FunctionComponent< Props > = ( props ) => {
 									 * Permission screen
 									 */
 									return <Title>You are not authorized to view this page</Title>;
+								} else if (
+									engine === 'medium' &&
+									isEnabled( 'gutenboarding/import-from-medium' )
+								) {
+									/**
+									 * Wix importer
```

Comment:
You probably mean `Medium` instead of `Wix`.

Reply:
Of course, it's a typo.